src/app/view/components/dot-iframe-dialog/dot-iframe-dialog.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit, Input, SimpleChanges, OnChanges, EventEmitter, Output } from '@angular/core';
+
+@Component({
+    selector: 'dot-iframe-dialog',
+    templateUrl: './dot-iframe-dialog.component.html',
+    styleUrls: ['./dot-iframe-dialog.component.scss']
+})
+export class DotIframeDialogComponent implements OnInit, OnChanges {
+    @Input() url: string;
+    @Output() close: EventEmitter<any> = new EventEmitter();
+    @Output() load: EventEmitter<any> = new EventEmitter();
+    @Output() keydown: EventEmitter<KeyboardEvent> = new EventEmitter();
+
+    dialogSize: any;
```

Comment:
maybe we should create a new interface here:

interface Size {
    width: number,
    height: number
}

Reply:
Getting rid of this.